packages/machine-extractor/src/toMachineConfig.ts

```diff
@@ -337,6 +337,10 @@ export const getTransitions = (
     if (transition?.description) {
       toPush.description = transition?.description.value;
     }
+    // XState transitions are external by default so only add `internal` if its true
```

Comment:
`target: '.child'` (without the explicit `internal` prop) will still be handled correctly further down the road, right?

Reply:
Can you give me an example and what's expected?

Reply:
The Studio should correctly set the external prop on its side to `external: false` for a transition defined like this `target: '.child'`. 

This also means that you should **always** extract `internal` property with boolean values and not only when it's set to `internal: true`. That is because this should be displayed as an external transition by the Studio:
```ts
target: '.child',
internal: false
```

Reply:
I haven't checked but I do believe if `internal` is missing, the Studio assumes it's an external transition.

Reply:
If the target's descriptor "kind" is ignored then this would be a pretty important bug to fix on the Studio's side. To fix it we'll definitely need to extract both `internal: false` and `internal: true` here.

Reply:
Let's stay on the safe side and always include the internal property.